docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,32 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build, do all of the above (except tauri dev) and:
+
+```sh
+cd ../src-tauri
+```
+
+```sh
+npm run tauri build
+```
```

Comment:
I'm a bit confused, are saying one should do this or not do this?

Reply:
Is this better?:
If you want to build, do the first two commands and:

Reply:
I mean say what to build, but yeah, ig